pkg/controller/add_tektoninstallation.go

```diff
@@ -0,0 +1,10 @@
+package controller
+
+import (
+	"github.com/codeready-toolchain/toolchain-operator/pkg/controller/tektoninstallation"
+)
+
+func init() {
+	// AddToManagerFuncs is a list of functions to create controllers and add them to a manager.
+	AddToManagerFuncs = append(AddToManagerFuncs, tektoninstallation.Add)
+}
```

Comment:
Suggestion: We can add directly into `controller.go` rather adding new file for each controller with one_line.  Example: https://github.com/nurali-techie/member-operator/blob/master/pkg/controller/controller.go

Reply:
Addressed in  64ab1f5